demo/MMSegmentation_Tutorial.ipynb

```diff
@@ -69,7 +69,8 @@
     "# Install PyTorch\n",
     "!conda install pytorch=1.10.0 torchvision cudatoolkit=11.1 -c pytorch\n",
     "# Install MMCV\n",
-    "!pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu101/torch1.6/index.html"
+    "#!pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu101/torch1.6/index.html\n",
+    "!pip install --upgrade mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu111/torch1.10/index.html"
```

Comment:
Might follow the official installation from mmcv 
```suggestion
    "!pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu111/torch1.10/index.html"
```
ref: https://github.com/open-mmlab/mmcv#installation